x-pack/platform/plugins/shared/alerting/server/data/ad_hoc_run/types/ad_hoc_run.ts

```diff
@@ -69,6 +73,8 @@ export interface AdHocRun {
   enabled: boolean;
```

Comment:
nit: It'd be great to add a comment explaining where we expect to use this interface. It's quite obvious `AdHocRunSO` matches the saved object but it's unclear whether `AdHocRun` types the domain model or an API response as well.